### DISCUSSION TITLE
Cannot Remap F13 / XF86Tools

### DISCUSSION THREAD

**Discussion by montanajava** [sezanzeb/input-remapper#760] [Q&A]
I wish to remap XF86Tools to a key sequence (XF86Tools appears to be mapped to my F13 key) and despite best efforts, it always brings up the Settings application. 

Any ideas?  I am using Fedora/Wayland.

**Comment by Tesla-Tank** [comment]
This is a copy and paste from reddit. I had to figure this out myself and managed to get this to work for both x11 and wayland.
The steps maybe different for you and even the files may not exist but I did this on Nobara 38 with no problems (On KDE).

Steps: You go to this directory "/usr/share/X11/xkb/symbols/inet" and open it with a text editor. Go to lines 193-204 and you'll see a bunch of FK13 to FK24 keys. Replace it with this: https://pastebin.com/uW5yW2eh (The steps are also there just incase). Restart and change the keybinds in settings.
